### ISSUE TITLE
Blocking call to open inside the event loop

### ISSUE THREAD

**Issue by illuzn** [HASwitchPlate/openHASP-custom-component#130]
## Version of the custom_component
0.7.3

## Configuration
Not Applicable

## Describe the bug
Can be fixed by a similar fix to #123. Similar Issue to #127.

When HA tries to restore entities it reaches this open call:
https://github.com/HASwitchPlate/openHASP-custom-component/blob/24f071cb510f3da0e5c801ccd52eb5abad9c4d6b/custom_components/openhasp/__init__.py#L353

## Debug log
```text
2024-07-03 18:24:41.528 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to open inside the event loop by custom integration 'openhasp' at custom_components/openhasp/__init__.py, line 353: with open( (offender: /config/custom_components/openhasp/__init__.py, line 353: with open(), please create a bug report at https://github.com/HASwitchPlate/openHASP-custom-component/issues
```

**Comment by dgomes** [comment]
this is literally the fix introduced in #123 it was supposed to have fixed :(

**Comment by illuzn** [comment]
This has disappeared in 2024.7.2 and I redownloaded the component a few times since.

Assume this was some bug/ error caused by me because I can no longer reproduce. Will open a new issue if it arises again.
